### DISCUSSION TITLE
Filtering order query with COT enabled

### DISCUSSION THREAD

**Discussion by Hug0-Drelon** [woocommerce/woocommerce#36559] [WooCommerce HPOS] [answered]
Hi 👋 

Is there any plan to implement a filter such as `woocommerce_order_data_store_cpt_get_orders_query` for order query using COT?

**Comment by vedanshujain** [comment, marked as answer]
hey, late response, but you can use the `woocommerce_orders_table_query_clauses` from WC 7.9 to achieve similar results.
